Document 'noWrapParams'

ankon opened the pull request:
I needed this functionality for https://github.com/vecmezoni/gulp-xslt/pull/2, but it took quite some code-reading until I found that node-libxslt already supports it.

albanm commented:
Thanks.

Could you run 'npm run docs' to generate the new README.md, then add it to this PR ?

ankon commented:
Sure, I'll try ...

Should this not be a step done by the travis/appveyor builds? :)

albanm commented:
Yes there is probably something to improve here !

But configuring the build to do a commit bothers me.

ankon commented:
Fair enough :)

`npm run docs` failed for me, because the `exported` partial seems to have been dropped, so I changed that to `members`. I also adjusted the other parts slightly so that the docs actually contain the type definitions. Should be ok now.

albanm commented:
Sorry, forgot to merge this earlier.
